### ISSUE TITLE
Add role specific category restriction to the categories filter in administration module

### ISSUE THREAD

**Issue by schmidt-oliver** [georgringer/news#461]
I can't find any way to restrict the categories filter in administration module.

In news plugin itself there is a simple way, for example over user tsconfig:
`page.TCEFORM.tx_news_domain_model_news.categories.config.treeConfig.rootUid=23`

Is there any way to restrict the categories filter in administration module to a given rootCategory seperated for different roles?

**Comment by georgringer** [comment]
you can use the category restricition in backend_user and backend_usergroups. currently nothing else supported. is that enough for you?

**Comment by schmidt-oliver** [comment]
> you can use the category restricition in backend_user and backend_usergroups. currently nothing else supported. is that enough for you?

Thanks for reply. First I'll give an example: There is currently one backend user group in our system that is allowed to use the news administration module. This user group is also allowed to use somewhere else categories in our system:

* News categories
  * News for homepage
  * News for students
  * News for researchers
  * News for ... (any other audience)
* Address categories
  * ...
* Content categories
* ...

Independent of used extensions the category restriction for content elements works pretty.
Based on the above example, you should see that a restriction over backend_user and backend_usergroups are ineligible.

Our news administrators know about the problem. For us, a limitation of the categories as described above is nice to have.

**Comment by georgringer** [comment]
sounds absolut valid. do you want to sponsor that?

**Comment by schmidt-oliver** [comment]
Currently not. How much budget do you calculate?

**Comment by georgringer** [comment]
sorry for the lack of feedback. fixed that now for upcomign 7.0 release. Docs for that are


```
Reduce the shown categories by defining a list of **root* category IDs.

Example:

.. code-block:: typoscript

    # Example category tree (value in brackets is the uid)
    [10] Cat 1
    [12] Cat 2
        [13] Cat 2 b
    [14] Cat 3
    [15] Cat 4

    tx_news.module.allowedCategoryRootIds = 12,15

    # Category tree shown
    [12] Cat 2
        [13] Cat 2 b
    [15] Cat 4
```
